src/common/IterationLookup.ts

```diff
@@ -265,13 +270,18 @@ export class Iterations implements IterationLookup {
    * (or today if no date is given).
    * @param {number} diff n for upcoming iterations, -n for previous iterations
    * @param {string|DateTime} [dateString] defaults to today
+   * @param {string} [baseIterationString] optionally provided iteration rather than date
```

Comment:
this line is too long. max line for comments is 80. download [this VS code extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) and bind its command to a keyboard shortcut like Cmd+Shift+R. then in your VS code settings JSON, add these:
```json
  "rewrap.wrappingColumn": 80,
  "editor.rulers": [80],
```
that will add a vertical ruler on the 80th column so you can clearly see when a comment is overflowing the limit, and it will set the limit to 80 for the rewrap extension.

you might also find this useful, if you use [an extension](https://marketplace.visualstudio.com/items?itemName=phoihos.git-commit-message-editor) to write commit messages:
```json
  "[git-commit]": {
    "rewrap.wrappingColumn": 72,
    "rewrap.autoWrap.enabled": true,
    "editor.rulers": [50, 72]
  }
```
that will give you rulers at 50 and 72 for commit messages specifically. which is useful since those are the limits for git commit messages - 50 for the first line (the short message) and 72 for subsequent lines (the extended description)

Reply:
Thanks, that's super useful!